Code/GraphMol/DistGeomHelpers/Wrap/testDistGeom.py

```diff
@@ -514,9 +517,20 @@ def testGithub1763(self):
     mol = Chem.MolFromSmiles('CCCCC')
     bm1 = rdDistGeom.GetMoleculeBoundsMatrix(mol)
     bm2 = rdDistGeom.GetMoleculeBoundsMatrix(mol,doTriangleSmoothing=False)
-    print(bm1)
-    print(bm2)
     self.assertTrue(bm1[0, 4] < bm2[0,4])
 
+  def testGithub2057(self):
+      # ensure that ETKDG is the default Embedder
+    mol = Chem.AddHs(Chem.MolFromSmiles('OCCC'))
+    fn = os.path.join(RDConfig.RDBaseDir, 'Code', 'GraphMol', 'DistGeomHelpers', 'test_data',
+                        'simple_torsion.etkdg.mol')
+    ref = Chem.MolFromMolFile(fn, removeHs=False)
+    params = rdDistGeom.EmbedParameters()
+    params.randomSeed = 42
+    params.useExpTorsionAnglePrefs = True
```

Comment:
Shouldn't useExpTorsionAnglePrefs and useBasicKnowledge default to True in EmbedParameters as well (to be fully consistent)?  This test indicates that they don't.

i.e. EmbedMolecule(mol, EmbedParameters()) should be the same as EmbedMolecule(mol)
Also, it doesn't appear that params are actually used here.

Reply:
There is an `ETKDG` parameter object, so I think it's ok that the default `EmbedParameters` object is simple/fast.

Reply:
This particular parameter object should be being crated at all; that's a mistake.